### ISSUE TITLE
Support for multiple statsd endpoints

### ISSUE THREAD

**Issue by reyjrar** [shutterstock/Net-Statsd-Client#5]
I started using your module, but I need to be able to spray stats to multiple collectors.  Currently, I'm wrapping your module, to do that it means the sample_rate bit is off from the beginning.

Two collectors get different results as due to the sample_rate check.  I got around this by creating an object foreach statsd endpoint, but sampling sampling is madness, so I override the sample_rate in those objects to 1.0 and my wrapper functions handle the sample rate check.  However, this means that I lose the sample_rate metric that's generated by statsite. 

I can submit a patch to support multiple end-points in your module so the sample_rate calculations aren't all borked up.  Is that something you'd be interested in merging/building/indexing on the CPAN?

Thanks!

**Comment by arodland** [comment]
absolutely would be happy to have it.

BTW please feel free to ignore the stuff about rockstack in the readme, it's dead and gone.

**Comment by reyjrar** [comment]
Cool,

I should have something today or tomorrow.  Thanks for the quick response!

**Comment by reyjrar** [comment]
Actually, I realized this needs to happen at the Etsy::StatsD layer.  I just opened a PR there for this feature, see: [Feature: Enable multiple statsd endpoints](https://github.com/sanbeg/Etsy-Statsd/pull/2)

I changing this with Net::Statsd::Client and passing an array reference through the host parameter to Etsy::StatsD and it behaved exactly as I expected.  I don't think there's any action needed on your part.

Thanks for following along with this insanity!
